client/components/web-preview/style.scss

```diff
@@ -207,17 +200,6 @@
 	}
 }
 
-.web-preview__toolbar-tray {
-	.button {
-		margin: 4px;
```

Comment:
This style seems needed as it can result in different `.button` elements have different 'heights'. 

Having said that, even if we add back this rule, in trunk the buttons and the left part of the toolbar have different heights. We should update to have the same.

Reply:
no, I'll find a different way to fix this. These margin are not the right way to position the buttons.